src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,8 @@
+// SIZES //
+$card-width: 200px;
+
+// COLORS //
```

Comment:
Comments are unnecessary here, since the variables speak for themselves about what is stored in them.